lib/routes/routePUT.js

```diff
@@ -143,10 +143,6 @@ export default function routePUT(request, response, log, utapi) {
                 return routesUtils.responseNoBody(errors.MissingContentLength,
                     null, response, 411, log);
             }
-            if (Number.isNaN(request.parsedContentLength)) {
-                return routesUtils.responseNoBody(errors.InvalidArgument,
```

Comment:
@LaurenSpiegel @DavidPineauScality I compared our errors to the Amazon ones, I updated our code to be as close as possible.